src/main/java/application/controller/LoginController.java

```diff
@@ -0,0 +1,45 @@
+package application.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import session.SessionManager;
+import webserver.RequestHandler;
+import webserver.annotation.Controller;
+import webserver.annotation.RequestMapping;
+import application.db.Database;
+import application.model.User;
+import http.request.HttpRequest;
+import http.response.HttpResponse;
+import util.HttpMethod;
+
+import java.util.Map;
+
+@Controller
+public class LoginController extends FrontController {
+    private static final Logger log = LoggerFactory.getLogger(RequestHandler.class);
+
+    @RequestMapping(path = "/users/login", method = HttpMethod.POST)
+    public String login(HttpRequest request
+            , HttpResponse response) {
+        Map<String, String> parameters = request.getHttpRequestBody().getBody();
+        log.debug("userId : {}", parameters.get("userId"));
+        log.debug("password : {}", parameters.get("password"));
+
+        String userId = parameters.get("userId");
+        String password = parameters.get("password");
+        User user = Database.findUserById(userId);
+        if (user.validUser(userId, password)) {
+            SessionManager.createSession(user, response);
+            response.addCookieHeader();
```

Comment:
요 부분 흐름이 조금 자연스럽지 않은 느낌입니다..
```suggestion
            String sid = SessionManager.createSession(user);            
            response.addCokie("sid", sid);
```
요 정도 어떨까요?